Vagrantfile

```diff
@@ -11,8 +11,9 @@ cd /vagrant
 grep -Fq 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
 
 # Install system packages
+export DEBIAN_FRONTEND=noninteractive
 sudo apt-get update --quiet
-sudo apt-get install -y $(cat debian_packages.lst) postgresql
+sudo -E apt-get install -y $(cat debian_packages.lst) postgresql mysql-server mongodb-server
```

Comment:
@omarkhan These additional dependencies would be missing for installations not using Vagrant.